### ISSUE TITLE
/flan devuelve el plan de estudios viejos

### ISSUE THREAD

**Issue by faustomartinez** [comcomUBA/dcubabot#74]
Calculo que se podría hacer un /flan1993 y un /flan2023 sin mucho problema jaja

**Comment by rozen03** [comment]
nah, considero q al gente del plan viejo ya sabe que hacer, yo creo q es hora de cambiar la imagen al plan nuevo

**Comment by faustomartinez** [comment]
tambien puede ser
